index.rst

```diff
@@ -10,7 +10,240 @@ This is complemented by the so called "run branch", which is a common branch use
 
 This technote describes the details of the Observing Environment and how it is managed by our team.
 
-Add content here
-================
+Introduction
+============
 
-See the `Documenteer documentation <https://documenteer.lsst.io/technotes/index.html>`_ for tips on how to write and configure your new technote.
+The distributed nature of the Vera Rubin Observatory Control System makes it challenging to manage shared application/library software packages between users and the control system.
+In order to mitigate this issue we have created to so called "observing environment".
+This is a collection of packages that are central for the observatory operation and spans components configurations, SAL Script and other user facing packages.
+The idea is that users and control system will have access to a centralized common collection of packages that can be centrally managed and audited.
```

Comment:
Consider rephrasing "both users and the control system". Also uses "centralized" and then "centrally". I think you can drop one of these.